apps/trading-e2e/src/integration/trading-deal-ticket-basics.cy.ts

```diff
@@ -81,7 +81,7 @@ describe(
           cy.visit('/#/markets/market-0');
         });
         it('must display that market is not accepting orders', function () {
-          cy.getByTestId('dealticket-error-message-summary').should(
+          cy.getByTestId('deal-ticket-error-message-summary').should(
```

Comment:
non blocking but do you think it's worth it to pull these selector strings out into a const. (same applies for the other test files?